addons/mail/static/src/components/message/message.scss

```diff
@@ -58,15 +56,21 @@
         border-bottom: 1px solid darken(map-get($grays, '400'), 5%);
     }
 
-    &.o-highlighted {
-        background-color: rgba($o-brand-primary, .1);
+    .o-isAuthorNotCurrentUserOrGuest {
+        opacity: .15;
+    }
 
-        &.o_Message_active {
-            background-color: rgba($o-brand-primary, .2);
-        }
+    .o_Message_prettyBody a.o_mail_redirect {
+        padding: map-get($spacers, 1);
+        border-radius: $border-radius;
+        background: rgba($primary, .1);
```

Comment:
unflatten, i.e. move it away from `.o_Message` rule and make it a top-level rule starting with `o_Message_prettyBody` immediately